lib/assessment.rb

```diff
@@ -21,13 +23,14 @@ def sum(enum)
   # in the numbers enumerable are odd
   def some_odd(numbers)
     # Your code goes here
+    numbers.reduce(false) {|memo, item| memo || item.odd?}
```

Comment:
Nice! I like seeing clever uses of `reduce`.
Best practice would probably be to do something like 
`numbers.any? {|num| num.odd?}`
or
`numbers.any?(&:odd?)`